docs/snippets/all/archetypes/depth_image_3d.cpp

```diff
@@ -31,7 +31,7 @@ int main() {
 
     rec.log(
         "world/camera/depth",
-        rerun::DepthImage({HEIGHT, WIDTH}, data)
+        rerun::DepthImage({WIDTH, HEIGHT}, data)
```

Comment:
TODO: change parameter orders here to force users to rethink their log code and read migration guide.

Also, "data, length" is our most common parameter order